src/components/Comments.jsx

```diff
@@ -32,6 +33,7 @@ function Comments({ article_id }) {
           </div>
         );
       })}
+      <CommentAdder article_id={article_id} setComments={setComments} />
```

Comment:
From a UI stand point the input box for adding a comment should ideally belong above the list of comments. 

As a user I should be able to comment on an article without having to scroll past all of the other comments on an article.